airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -27,6 +27,7 @@
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 from airflow.hooks.dbapi_hook import DbApiHook
 from apiclient.discovery import build
+from apiclient.discovery import HttpError
```

Comment:
This line can be combined with line above:

    from apiclient.discovery import build, HttpError